Makefile

```diff
@@ -74,13 +74,12 @@ docs: ## generate Sphinx HTML documentation, including API docs
 servedocs: docs ## compile the docs watching for changes
 	watchmedo shell-command -p '*.rst' -c '$(MAKE) -C docs html' -R -D .
 
-release: clean ## package and upload a release
-	python setup.py sdist upload
-	python setup.py bdist_wheel upload
+release: clean ## package and upload a release (requires a `~/.pypirc` file)
+	twine upload dist/*
```

Comment:
I prefer to fall back on `twine`. `twine` prompts for the username and password if the default config file is not found, e.g.:

```bash
$ twine upload dummy
Uploading distributions to https://upload.pypi.org/legacy/
Enter your username: 
# ...
```